packages/aries-hooks/src/index.tsx

```diff
@@ -0,0 +1,295 @@
+import React, { createContext, useState, useEffect, useContext } from 'react'
+
+import { downloadGenesis, storeGenesis } from '../genesis-utils'
+
+import {
+  Agent,
+  InitConfig,
+  HttpOutboundTransport,
+  ConnectionState,
+  CredentialState,
+  ProofState,
+  ConnectionEventTypes,
+  CredentialEventTypes,
+  ProofEventTypes,
+  ConnectionStateChangedEvent,
+  CredentialStateChangedEvent,
+  ProofStateChangedEvent,
+  ConnectionRecord,
+  ProofRecord,
+  CredentialRecord,
+  WsOutboundTransport,
+  BasicMessageRecord,
+  BasicMessageReceivedEvent,
+  BasicMessageEventTypes,
+} from '@aries-framework/core'
+
+import { agentDependencies } from '@aries-framework/react-native'
+
+const TAG = 'Aries-Hooks:'
```

Comment:
I do like the idea of prefixing the logs with some indicator like this. However I am not sure if this is the way to go. Would like to hear other opinions about this.